server/sonar-db-dao/src/main/java/org/sonar/db/qualityprofile/QualityProfileMapper.java

```diff
@@ -32,29 +32,22 @@
 
   void delete(int id);
 
-  List<QualityProfileDto> selectAll();
-
-  @CheckForNull
-  QualityProfileDto selectDefaultProfile(@Param("language") String language);
-
-  List<QualityProfileDto> selectDefaultProfiles(@Param("languages") List<String> languages);
-
   /**
    * @deprecated provide organization
    */
   @Deprecated
+  List<QualityProfileDto> selectAll();
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png 'Severity: INFO') Do not forget to remove this deprecated code someday. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1133)